### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by marcolink**
Add hint for troubleshooting too big response size:

[rendered Readme](https://github.com/contentful/contentful-export/blob/c46f107834bd4e112c2aaba6c9dab8df15a4df94/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,16 +223,32 @@ Display progress in new lines instead of displaying a busy spinner and the statu
...
+contentfulExport({
+  proxy: 'https://cat:dog@example.com:1234',
+  rawProxy: true,
+  ...
+})
+```
+
+### Error: 400 - Bad Request - Response size too big.
+
+Lower the amount of entites received within a single request by setting the maxAllowedLimit option:
```
Comment (stefanjudis) [inline README.md:240]: @marcolink could we link out do docs, describing what this error means, too?

Thinking of something along the lines "Contentful response sizes are limited (find more info in our technical limit docs). To resolve this issue, limit the queried entries" or something. 🙈

Comment (marcolink) [reply README.md]: @stefanjudis good point - please review the updated version!

File: README.md [hunk 2]
```diff
@@ -223,16 +223,32 @@ Display progress in new lines instead of displaying a busy spinner and the statu
...
+contentfulExport({
+  proxy: 'https://cat:dog@example.com:1234',
+  rawProxy: true,
+  ...
+})
+```
+
+### Error: 400 - Bad Request - Response size too big.
+
+Contentful response sizes are limited (find more info in our [technical limit docs](https://www.contentful.com/developers/docs/technical-limits/)). In order to resolve this issue, limit the amount of entites received within a single request by setting the [`maxAllowedLimit`](#maxallowedlimit-number-default-1000) option:
```
Comment (stefanjudis) [inline README.md:240]:
```suggestion
Contentful response sizes are limited (find more info in our [technical limit docs](https://www.contentful.com/developers/docs/technical-limits/)). In order to resolve this issue, limit the amount of entities received within a single request by setting the [`maxAllowedLimit`](#maxallowedlimit-number-default-1000) option:
```

**Review by stefanjudis — APPROVED** [review]
Tiny type, but apart from that 🚢

**Pull request merged**